pkg/didcomm/packager/packager.go

```diff
@@ -146,10 +150,21 @@ func (bp *Packager) UnpackMessage(encMessage []byte) (*transport.Envelope, error
 		return nil, fmt.Errorf("message Type not recognized")
 	}
 
-	data, senderVerKey, err := p.Unpack(encMessage)
+	envelope, err := p.Unpack(encMessage)
 	if err != nil {
 		return nil, fmt.Errorf("unpack: %w", err)
 	}
 
-	return &transport.Envelope{Message: data, FromVerKey: base58.Encode(senderVerKey)}, nil
+	//	ignore error - agents can communicate without using DIDs - for example, in DIDExchange
+	// nolint: errcheck
+	theirDID, _ := bp.connectionStore.GetDID(base58.Encode(envelope.FromVerKey))
```

Comment:
Suspicious about not even examining the error for the allowed errors in this situation.
Similarly with L164.

Reply:
This is done now.